Resources/public/js/views/fields/ez-emailaddress-editview.js

```diff
@@ -101,6 +101,21 @@ YUI.add('ez-emailaddress-editview', function (Y) {
         _getFieldValue: function () {
             return this.get('container').one('.ez-emailaddress-input-ui input').get('value');
         },
+
+        /**
+         * Returns email address format validity
```

Comment:
I would write something like:
```
Checks email address validity based on the same regexp as
the one used in the EmailAddress FieldType.
Otherwise, the email address edit view could accept email
that will be considered invalid when creating/updating a content.
```